pom.xml

```diff
@@ -19,9 +19,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <build-support.version>1.4</build-support.version>
     <checkstyle.version>8.20</checkstyle.version>
-    <slf4j.version>1.8.0-beta2</slf4j.version>
-    <log4j2.version>2.11.2</log4j2.version>
-    <logback.version>1.3.0-alpha4</logback.version>
+    <slf4j.version>1.8.0-beta1</slf4j.version>
```

Comment:
Why we want back to 1.8.0-beta1 from 1.8.0-beta2?

Reply:
Can we do this work against 2.0.0-alpha0.  I know that is an early release, but there is no point going to 1.8 for this change.  If we work against the alpha then we can see any problems and give feedback

Reply:
While slf4j-api `2.0.0-alpha0` exists, there's no public (or SNAPSHOT) release of `log4j2` that uses it (yet).
That's the only reason I stuck with 1.8.0

Reply:
I'd probably go with `2.0.0-alpha1` if you are ok with no log4j2 support (yet)

Reply:
We know we will want to go to 2 as soon as it is available, so I think working with 1.8 is not really productive.
I'd much rather find out problems with 2 now when we have a chance to influence the design.

Reply:
Updated to `2.0.0-alpha1`